src/document.c

```diff
@@ -695,7 +689,7 @@ FIELD_PREPROCESSOR(geoPreprocessor) {
     case FLD_VAR_T_BLOB_ARRAY:
     case FLD_VAR_T_NUM:
     case FLD_VAR_T_GEOMETRY:
-      RS_LOG_ASSERT(0, "Oops");
+      RS_LOG_ASSERT(0, "Unsupported field type GEOMETRIY for GEO index");
```

Comment:
Could also be `FLD_VAR_T_NUM` or `FLD_VAR_T_BLOB_ARRAY`
```suggestion
      RS_LOG_ASSERT(0, "Unsupported field type for GEO index");
```